src/components/Profile/components/PersonalInfoList/index.jsx

```diff
@@ -26,9 +29,29 @@ import UpdatePhone from './components/UpdatePhoneDialog';
 import styles from './PersonalInfoList.module.css';
 import AlumniUpdateForm from './components/AlumniUpdateForm';
 import CliftonStrengthsService from 'services/cliftonStrengths';
+import SLock from './Salsbury.png';
+import DPLock from './DandP.png';
+import DDLock from './DandD.png';
+import useWindowSize from 'hooks/useWindowSize';
 
 const PRIVATE_INFO = 'Private as requested.';
 
+// const Lock1 = width >= 200 ? : SLock;
+
+// const Lock2 = width >= 200 ? : DPLock;
+
+// const Lock3 = width >= 200 ? DDLock;
+
+const CustomTooltip = withStyles((theme) => ({
+  tooltip: {
+    backgroundColor: theme.palette.common.black,
+    color: 'rgba(255, 255, 255, 0.87)',
```

Comment:
Hard-coding colors into your code makes it hard to maintain in the long term. Check out `theme.ts` for the colors that are available to you.